src/fn/README.md

```diff
@@ -0,0 +1,26 @@
+# Actions
+
+Actions are defined as usable if they exist in this directory.
+
+The API is that they should export something like this:
+
+```js
+module.exports = {
+    requiresInstance: true,
+    execute: async (projectDir, couchUrl, extraArgs) => {
```

Comment:
It seems you call this `execute()` here in the documents but `action()` in the code. I prefer your choice of execute. A little confusing to use `action` to describe both the files and a function within them.

Reply:
Whoops, yeah it's supposed to be `execute`, I'll change it